mplc/multi_partner_learning.py

```diff
@@ -654,3 +655,153 @@ def init_multi_partner_learning_from_scenario(scenario, is_save_data=True):
     )
 
     return mpl
+
+
+class MplLabelFlip(MultiPartnerLearning):
```

Comment:
add link to paper / blog post

Reply:
There is no paper yet, nor blog post. Thomas and I are working on the paper.